cmd/containerd/command/main.go

```diff
@@ -183,10 +183,10 @@ can be used and modified as necessary as a custom configuration.`
 			err error
 		}
 
-		// run server initialization in a goroutine so we don't end up blocking important things like SIGTERM handling
+		// run server initialization in a goroutine, so we don't end up blocking important things like SIGTERM handling
```

Comment:
Can you revert this change?  The previous text is correct; no comma is necessary.